storage/src/main/java/com/tencent/rss/storage/handler/impl/ComposedClientReadHandler.java

```diff
@@ -55,32 +83,55 @@ public ShuffleDataResult readShuffleData() {
     switch (currentHandler) {
       case HOT:
         try {
+          if (hotDataReadHandler == null) {
```

Comment:
The method to check & create handler can be wrapped as a method

Reply:
OK